tests/boot/boot_from_pxe.pm

```diff
@@ -96,7 +96,7 @@ sub run {
         $image_path .= 'plymouth.enable=0 ';
     }
     # Execute installation command on pxe management cmd console
-    type_string_slow ${image_path};
+    type_string_slow ${image_path} . " ";
```

Comment:
Why not to set default value to $image_path as ``" "`` ?
So we would avoid undef.
```
[2019-01-11T07:51:30.997 CET] [debug] /var/lib/openqa/cache/openqa.suse.de/tests/sle/tests/boot/boot_from_pxe.pm:99 called utils::type_string_slow
[2019-01-11T07:51:30.997 CET] [debug] <<< testapi::type_string(string=undef, max_interval=13, wait_screen_changes=0, wait_still_screen=0)
```
Otherwise LGTM

Reply:
I wont be undef with space same as it was [before](https://openqa.suse.de/tests/2360502/file/autoinst-log.txt),
Well it can be like this or I can do `$image_path=' '`

Reply:
hm...nice! Fantastic perl!